tests/test_dice_loss.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2020 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+import torch
+from parameterized import parameterized
+
+from monai.networks.losses.dice import DiceLoss
+
+
+class TestDiceLoss(unittest.TestCase):
+
+    @parameterized.expand([
+        [
+            {
+                'include_background': False,
+            },
+            {
+                'pred': torch.tensor([[[[1., -1.], [-1., 1.]]]]),
+                'ground': torch.tensor([[[[1., 0.], [1., 1.]]]]),
+                'smooth': 1e-6,
+            },
+            0.307576,
+        ],
+        [
+            {
+                'include_background': True,
+            },
+            {
+                'pred': torch.tensor([[[[1., -1.], [-1., 1.]]], [[[1., -1.], [-1., 1.]]]]),
+                'ground': torch.tensor([[[[1., 1.], [1., 1.]]], [[[1., 0.], [1., 0.]]]]),
+                'smooth': 1e-4,
+            },
+            0.416636,
+        ],
+    ])
```

Comment:
This looks like a good way to write multiple tests using the same code, so long as it plays well with the output from unittest when they fail. Perhaps put this list into a separate variable above the test class however just so the @parameterized tag doesn't get lost.